recipes/adc-streaming/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "adc-streaming" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "c1ab8aee52684ff1f1c5ae469a760a435d6c45836ead4e9a97423a4893cb92d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/adc-streaming-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
This is a pure Python package and should be defined as [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds) instead of skipping windows: `noarch: python`

Reply:
I had done this due to non-existent Windows packages for python-confluent-kafka, and removing this causes the win64 stage to fail. Given this, is defining `noarch: python` still acceptable?

Reply:
Yes, windows build will fail in the staged-recipe repo because build is always done for all platforms here.
But `noarch` packages are only built on Linux once accepted.

If a user tries to install it on windows, conda will display a helpful error message describing which dependency couldn’t be found.

Reply:
Okay, great.